### ISSUE TITLE
Adding "..." (the three dots) to archive/post element text excerpts

### ISSUE THREAD

**Issue by Luxamman** [pojome/elementor#6399]
https://github.com/pojome/elementor/issues/4440 
https://github.com/pojome/elementor/issues/4499

**What problem is your feature request going to solve? Please describe.**
Text excerpts are looking like "cut" because there are no "..." dots at the end of the text, like people are used to have. 

**Describe the solution you'd like**
Adding a option to add three dots on archive and post elements excerpts. 

**Describe alternatives you've considered**
Display the dots all the time, except when the post has a manual written excerpts text.

**Comment by bainternet** [comment]
@Luxamman 	

If you are talking about the posts widget, then it uses the WP native `the_excerpt` which has its own filters that you can use ex:

```PHP
add_filter( 'excerpt_more', function( $more ) {
	return '...';
}, 30 );
```

thanks

**Comment by AlexDeat** [comment]
The "Archive Posts" Element is the issue for me, it doesnt have the 3 dots at the end, how do i manage to solve this?

**Comment by AlexDeat** [comment]
i managed by doing this

p:after {
    content: "...";
}

**Comment by thechiefalone** [comment]
> i managed by doing this
> 
> p:after {
> content: "...";
> }

I also added css to the Hello Child theme : 

/* 3 Dots after excerpt */

.elementor-post__excerpt p:after{
    content: "...";
}

**Comment by wassim-sayerh** [comment]
> i managed by doing this
> 
> p:after {
> content: "...";
> }

Excellent, thank you.

**Comment by get8bit** [comment]
Somebody dropped the ball on this one. Ellipses at the end of an excerpt is day one stuff, Elementor. This should be default.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.13.0! 🥳 

You can now use the Loop Builder to add the Post Excerpt widget and include the ellipsis at the end of the Post Excerpt.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
